opentelemetry-java-instrumentation.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: opentelemetry-java-instrumentation
+  version: "2.16.0"
+  epoch: 0
+  description: OpenTelemetry auto-instrumentation and instrumentation libraries for Java
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - busybox
+  target-architecture:
+    - x86_64
```

Comment:
Can we drop a comment why its only enabled for single arch?

Reply:
Yes, thanks for pointing this out!